h2o-bindings/bin/bindings.py

```diff
@@ -360,23 +360,30 @@ def find_field(fields, field_name):
     if add_generics:
         for base, generics in [
             # Note: derived classes must come before base classes here
-            ("SharedTreeModelV3", [("P", "ModelParametersSchemaV3"), ("O", "ModelOutputSchemaV3")]),
-            ("ModelSchemaV3", [("P", "ModelParametersSchemaV3"), ("O", "ModelOutputSchemaV3")]),
+            ("SharedTreeModelV3", [("P", "ModelParametersSchemaV3"), ("PS", "ModelParametersSchemaV3"), ("O", "ModelOutputSchemaV3")]),
+            ("ModelSchemaV3", [("P", "ModelParametersSchemaV3"), ("PS", "ModelParametersSchemaV3"), ("O", "ModelOutputSchemaV3")]),
             ("SharedTreeV3", [("P", "ModelParametersSchemaV3")]),
             ("ModelBuilderSchema", [("P", "ModelParametersSchemaV3")]),
         ]:
             # Write the generic information about the base class
             schema = m[base]
             schema["generics"] = generics
-            generic_map = {long_type: gen_type for gen_type, long_type in generics}
+            generic_map = DictList()
```

Comment:
besides, with `default_dict` you see immediately what kind of structure you're manipulating:
```
generic_map[schema_name].append(name)
```
instead of
```
generic_map[schema_name] = name
```
the latter requiring you to know that the datastructure overrides `__setitem__`, which should be done only when "necessary"

Reply:
f75f55b